packages/react-instantsearch-core/src/core/__tests__/indexUtils.js

```diff
@@ -419,13 +405,32 @@ describe('utility method for manipulating the search state', () => {
         {},
         context,
         'refinement',
-        null,
-        () => {}
+        null
       );
 
       expect(value).toEqual('defaultRefinement');
     });
 
+    it('retrieve the current refinement value when impossible', () => {
```

Comment:
I didn't understood the purpose of the test at first. We should mention the default value, WDYT?